### DISCUSSION TITLE
Is there anywhere to check what channel id that bot is connecting?

### DISCUSSION THREAD

**Discussion by zvyap** [lavalink-devs/lavalink-client#10] [Q&A] [answered]
Is there anyways to check what channel id that bot is connecting to or check is user is connecting to the same voice channel as the bot

Im not sure whether `sessionId` of the voice state can do this trick or not, but I really dont want to enable `voice state event ` for my bot client for performance issue.

**Comment by topi314** [comment]
I'm honestly not sure what you're asking here, but there is no way around the voice states intent for music bots.

You should also discuss this with your discord lib devs instead

**Reply by zvyap** [reply]
For clarify, Im asking whether I can get Lavalink connected audio channel id with JDA

**Comment by duncte123** [comment, marked as answer]
if you're using JDA: `guild.getSelfMember().getVoiceState()` will give you the channel that you are currently connected to.

Anything that requires checking about what channel a user is should be part of the discord lib you are using. The lavalink client only helps interacting with lavalink, and lavalink doesn't know what users are in what voice channel.

**Reply by zvyap** [reply]
oh! If Im not misunderstanding I can just use `guild.getSelfMember().getVoiceState()` as normal with Lavalink
Thx a lot

**Reply by duncte123** [reply]
Correct, it says that right here in the readme https://github.com/lavalink-devs/lavalink-client#installation-and-usage-with-jda

**Reply by zvyap** [reply]
I thought the selfmember voice state will not be updated because I use Lavalink to handle voice, my misunderstanding

Thx
